windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -17,23 +17,43 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
 
 You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
-[id="wmco-1-0-2"]
-== Release notes for Red Hat Windows Machine Config Operator 1.0.2
+[id="wmco-2-0-0"]
+== Release notes for Red Hat Windows Machine Config Operator 2.0.0
 
-This release of the WMCO provides initial support for running Windows compute nodes in an {product-title} cluster.
+This release of the WMCO provides bug fixes and enhancements for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 2.0.0 were released in link:https://access.redhat.com/errata/RHBA-2021:0440[RHBA-2021:0440].
 
 WMCO supports self-managed clusters built using installer-provisioned infrastructure running on the following cloud providers:
 
 * Amazon Web Services (AWS)
 * Microsoft Azure
+* VMware vSphere
 
-The following Windows Server operating systems are supported in the initial release of the WMCO:
+The following Windows Server operating systems are supported in this release of the WMCO:
 
 * Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
 
-[id="wmco-1-0-1-known-issues"]
+The WMCO 2.x version is only compatible with {product-title} 4.7.
+
+[id="wmco-2-0-0-enhancements"]
+=== New features and enhancements
+
+This release adds the following new features and improvements.
+
+[id="wmco-2-0-0-vsphere-support"]
+==== Support for clusters running on VMware vSphere
```

Comment:
I suggest calling the vSphere versions here. This would be whatever OpenShift support for vSphere IPI.

Reply:
@aravindhp our vSphere versions we support are [here](https://docs.openshift.com/container-platform/4.6/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-vsphere-infrastructure_installing-vsphere-installer-provisioned). We have a generic support statement for vSphere versions 6 and 7, followed by specific versions based on vSphere component that will be used.

Is a generic *vSphere version 6 or 7* support statement enough for the WINC use case? Or should there be a bit more detail provided?

Reply:
I am fine with the generic statement.

Reply:
I've added vSphere versions 6.5, 6.7, and 7.0 based on Slack conversation with QE.